### PR TITLE
custom html title for `Handout` class

### PR DESCRIPTION
**Pull request by epogrebnyak**
Added argument for custom html document title in `Handout` class  and proposed unit test layout.

### CODE REVIEW

**Comment by danijar** [comment]
Thanks! To answer your question from #13, pytest is good. This doesn't change your test besides that you can use the `tmp_path` fixture.

Small nit: let's keep using single quote for consistency.

**Review by danijar — CHANGES_REQUESTED** [review]
Looks good. Please switch double to single quotes for consistency and use the `tmp_path` pytest fixture, i.e.:

```
import handout

 def test_handout_on_title_arg_inserts_title(tmp_path):
    output = handout.Handout(tmp_path, title='This string')._generate(source='')   
    assert '<title>This string</title>' in output 
```

File: handout/handout.py [hunk 1]
```diff
@@ -10,9 +10,10 @@
 
 class Handout(object):
 
-  def __init__(self, directory):
+  def __init__(self, directory, title="Handout"):
```
Comment (danijar) [inline handout/handout.py]: single quotes please

**Review by danijar — CHANGES_REQUESTED** [review]

**Comment by epogrebnyak** [comment]
Sorry adopting single quote was slow. Hope I'd be able to contribute to new PRs faster.

**Comment by danijar** [comment]
No worries, thanks for the PR and your patience! :)

**Pull request merged**